btree/btree.c

```diff
@@ -1847,6 +1853,107 @@ M0_INTERNAL void m0_btree_mod_fini(void)
 	m0_free(mod_get());
 }
 
+M0_INTERNAL void m0_btree_lrulist_set_wm(int64_t slow_lru_mem_release,
+					 int64_t wm_low, int64_t wm_target,
+					 int64_t wm_high)
+{
+	if (slow_lru_mem_release != -1)
+		set_lrulist_trickle_release = slow_lru_mem_release;
+	else
+		set_lrulist_trickle_release = 0;
```

Comment:
```suggestion
	set_lrulist_trickle_release = (slow_lru_mem_release != -1) ? true : false;
```

Reply:
```
lrulist_trickle_release = (slow_lru_mem_release != -1 && slow_lru_mem_release != 0) ? true : false;
```

Reply:
Done. 
Since we are not passing -1 for incorrect values in sysconfig/motr, I have made the change as needed.